bigbluebutton-html5/imports/api/audio/client/bridge/service.js

```diff
@@ -24,6 +24,17 @@ const getCurrentAudioSessionNumber = () => sessionStorage.getItem(AUDIO_SESSION_
 const reloadAudioElement = (audioElement) => {
   if (audioElement && (audioElement.readyState > 0)) {
     audioElement.load();
+    if (audioElement.paused) {
+      audioElement.play().catch((error) => {
+        logger.error({
+          logCode: 'audio_element_play_error',
```

Comment:
```suggestion
          logCode: 'audio_reload_element_play_error',
```

logCode should be indicative of the caller as element playing is called in multiple places in the codebase.